src/lib/Libnet/net_server.c

```diff
@@ -549,26 +608,20 @@ wait_request(time_t waittime)
 				}
 			}
 #endif
-			idx = connection_find_actual_index(em_fd);
-			if (idx < 0) {
-				return -1;
+			j = 0;
+			if (scks && scks->active_socks && prio_sock_processed) {
```

Comment:
You can change the condition as follows.
if (scks && prio_sock_processed && scks->active_socks && )

Reply:
If there are not active sockets at all there is no need to check if its even processed . So keeping it the same . I'll change the next condition.